client-hc/src/main/java/com/graphhopper/api/GraphHopperMatrixWeb.java

```diff
@@ -38,9 +38,8 @@ public GraphHopperMatrixWeb setKey(String key) {
     public MatrixResponse route(GHMRequest request) {
         if (!Helper.isEmpty(key))
             request.getHints().putObject(KEY, key);
-        if (!request.getPathDetails().isEmpty())
-            throw new IllegalArgumentException("Path details are not supported for the Matrix API");
-        request.compactPointHints();
+        // todonow
+//        request.compactPointHints();
```

Comment:
probably there is nothing we need to compact anymore, because the point_hint field will simply be missing when it is not used?